tutorial/eventrelatedaveraging.md

```diff
@@ -119,8 +120,8 @@ The output is the data structure avgFIC with the following field
     avgFIC =
        time: [1x900 double]
       label: {149x1 cell
-       elec: [1×1 struct]
-       grad: [1×1 struct]
+       elec: [1x1 struct]
```

Comment:
Nice, thanks. 

There is this unicode character "×" used in MATLAB which does not show up so nicely on the website. That is why I always replace the "×" with an "x" from the keyboard. After all tutorial changes are done, we might again want to do a general search and replace.